modules/core/core-app/src/main/java/com/enonic/xp/core/impl/app/ApplicationFactory.java

```diff
@@ -78,6 +79,21 @@ ApplicationUrlResolver createUrlResolver( final Bundle bundle )
         }
     }
 
+    ApplicationUrlResolver createUrlResolver( final Bundle bundle, final String name )
+    {
+        switch ( name )
```

Comment:
we are missing one which is `new MultiApplicationUrlResolver( classLoaderUrlResolver, bundleUrlResolver )`. Let's call it "real" for now